README.md

```diff
@@ -28,6 +29,22 @@ If there was no exporter configured or they haven't exported for a while it get
 
 In order to shed some more light in the dark we build a tool called zdb - Zeebe Debugger. It should help you along the way during incidents and broken systems.
 
+
+## Installation
+
+Substitute `<version>` with the version you want to install. Check [releases](https://github.com/Zelldon/zdb/releases) to see all the available versions.
+```
+$ docker exec -it <container_id> /bin/bash
+$ cd /usr/bin
+$ apt-get update
+$ apt-get install curl
```

Comment:
Not all user run zeebe in a container. I think this can be replaced with something like: 'requires curl'

Reply:
Updated this.